country-info.csv

```diff
@@ -226,7 +226,8 @@ Tampere University,europe,fi
 Tata Inst. of Fundamental Research,asia,in
 Technion,europe,il
 Tel Aviv University,europe,il
-Texas A&M at Qatar,asia,qa
+Texas A&M at Qatar,asia,qa
+The Catholic University of Korea,asia,kr

```

Comment:
> If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

Does this PR (5 people) contain the entire faculty?

Reply:
yes, they are all.